### ISSUE TITLE
meiotic DNA replication

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#16509]
Hello,

GO:1902974 meiotic DNA replication initiation
Def: Any DNA replication initiation involved in meiotic cell cycle DNA replication. 

GO:0072691 initiation of premeiotic DNA replication
Def: A process of DNA-dependent DNA replication initiation that occurs as part of premeiotic DNA replication.

GO:0006279 premeiotic DNA replication
Def:  The replication of DNA that precedes meiotic cell division. 

GO:0072691 is_a GO:1902974
GO:0072691 is_a GO:0006279

but is GO:1902974 needed? Are there organisms that rereplicate their genome after nuclear divisions begin?

**Comment by pgaudet** [comment]
There are 5 pombase EXP annotations - what does that term represent that is distinct from its parent ?

**Comment by Antonialock** [comment]
I'm not sure what you mean by 'what does that term represent that is distinct from its parent' (which term?) 

My point was can GO:1902974 and GO:0072691 merge? (I don't know of any organisms that replicate their DNA after meiosis I is inititated)

**Comment by pgaudet** [comment]
The branch seems to have several unnecessary terms: 

> 'DNA replication initiation'
>> 'cell cycle DNA replication initiation'
>>> 'bacterial-type DNA replication initiation' -> 2 annotations; other bacterial genes are annotated to 'DNA replication initiation' ir its regulation
>>> 'nuclear cell cycle DNA replication initiation'
>>>> 'meiotic DNA replication initiation'
>>>>'mitotic DNA replication initiation'

It seems to me all we need is 

> 'DNA replication initiation'
>> 'meiotic DNA replication initiation'
>>'mitotic DNA replication initiation'

@ValWood @Antonialock What do you think ? 

Thanks, Pascale

**Comment by ValWood** [comment]
This would be more a question for @mah11 

I IRC the only reason why we have mitotic and meiotic DNA replication historically is to get these processes under mitotic and meiotic cell cycle, so that if you did a query on "mitotic cell cycle" you would retrieve everything involved. I now think this is  impractical solution. Probably  better to have

mitotic cell cycle
has_part  DNA replication and figure a way to make queries work  through the correct inferences?

The upstream regulation might be a little different, but the processes are absolutely the same.

**Comment by mah11** [comment]
I agree that there seem to be more terms than are truly necessary, but I think the important distinction is between bacterial-type and, well, the rest. Bacterial initiation is very different at the molecular level (quite a bit simpler) from initiation in eukaryotic nuclear DNA. Of the existing terms, therefore, these are the ones that represent genuinely different processes:

> 'DNA replication initiation'
>>
>>  'bacterial-type DNA replication initiation'
>>  'nuclear DNA replication initiation'

... but there is another twist: archaeal replication is more similar to eukaryotic than bacterial (the current thinking is that euks got replication from the archael side of their ancestry), but isn't "nuclear" in the literal sense. I don't remember how mitochondrial or plastid initiation work, but given the evolutionary origin of the organelles I wouldn't be surprised if they resemble the bacterial process.

In eukaryotes initiation is the same in (the rounds of DNA replication preceding) mitosis and meiosis, so the distinction between 'meiotic DNA replication initiation' and 'mitotic DNA replication initiation' is only one of timing relative to other cell cycle events, not any of the molecular events of initiation itself.

I see no meaningful difference whatsoever between GO:1902974 and GO:0072691.

The pombe annotations (4 now) are all to GO:1904514 'positive regulation of initiation of premeiotic DNA replication'. That's actually fine -- this is another case where one process can be differentially regulated in different contexts (another example was also with DNA replication initiation, regulated differently at different origins; see #17680 and #12533). So the meiosis-specific regulation terms should be retained, but the 'meiotic DNA replication initiation' terms are only useful if there's no other way to craft logical definitions for the regulation terms (and in that case, merge GO:1902974 and GO:0072691).
